README.md

```diff
@@ -96,6 +96,21 @@ Run all the tests (unittests only currently) with:
 ```sh
 tox
 ```
+## Proxy Support
+There are 2 ways you can use an HTTP(S) proxy. The first method is using environment variables:
```

Comment:
Actually another comment here, I would state that this is actually for the `requests` library and their documentation is where to lookup how this works. It's not obvious from this comment that the code will ever use the environment variables (the code doesn't look them up).